src/Microsoft.Azure.SignalR.Common/Endpoints/AadAccessKey.cs

```diff
@@ -61,14 +61,10 @@ private set
 
         private Task<object> InitializedTask => _initializedTcs.Task;
 
-        public AadAccessKey(Uri uri, TokenCredential credential) : base(uri)
+        public AadAccessKey(Uri endpoint, TokenCredential credential, Uri serverEndpoint = null) : base(endpoint)
         {
-            var builder = new UriBuilder(Endpoint)
-            {
-                Path = "/api/v1/auth/accessKey",
-                Port = uri.Port
-            };
-            AuthorizeUrl = builder.Uri.AbsoluteUri;
+            var authorizeUri = (serverEndpoint ?? endpoint).Append("/api/v1/auth/accessKey");
```

Comment:
is Port missing?

Reply:
Port should be included in `endpoint` or `serverEndpoint`